tests/optim/test_oss.py

```diff
@@ -564,6 +564,7 @@ def closure():
     dist.destroy_process_group(process_group)
 
 
+@skip_if_py39_no_cuda
 def test_multiple_groups():
```

Comment:
the failure case is specifically on agents which do not have cuda, even if this should not matter to this specific test (gloo and cpu).